docs/source/models/photons.rst

```diff
@@ -0,0 +1,50 @@
+Photons
+=======
+
+Radiation reaction and (hard) photons: why and when are they needed.
+Models we implemented and verified:
+
+* Landau-Lifschitz Model (semi-classical)
```

Comment:
The Reduced Landau Lifschitz Model (RLL) is actually a particle pusher and does not support photons, thus I would not place it under *photon models*.